Resources/Prototypes/Entities/Objects/Weapons/Guns/turrets.yml

```
@@ -128,6 +128,48 @@
+        path: /Audio/Weapons/Guns/Gunshots/c-20r.ogg
+    - type: BallisticAmmoProvider
+      proto: CartridgePistol
+      capacity: 200
+    - type: Destructible
+      thresholds:
+        - trigger:
+            !type:DamageTrigger
+            damage: 200
+          behaviors:
+            - !type:DoActsBehavior
+              acts: [ "Destruction" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)